internal/tpm/nosimulator.go

```diff
@@ -0,0 +1,9 @@
+//go:build !(amd64 || arm64)
+
+package tpm
+
+import "errors"
+
+func OpenTPMSimulator() (*TPM, error) {
+	return &TPM{}, errors.New("TPM simulator not supported")
```

Comment:
Doesnt this mean tests will always fail for other distros

Reply:
We currently skip the event log test if the event log isn't available, so we could do similar for TPM in general.

Reply:
Makes sense vs perm fail.